<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmWaveLine</class>
 <widget class="QWidget" name="frmWaveLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="WaveLine" name="waveLine1"/>
   </item>
   <item>
    <widget class="WaveLine" name="waveLine2"/>
   </item>
   <item>
    <widget class="WaveLine" name="waveLine3"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WaveLine</class>
   <extends>QWidget</extends>
   <header>waveline.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
